src/common/utilities.ts

```diff
@@ -176,3 +176,21 @@ export async function updateKedroVizPanel(lsClient: LanguageClient | undefined):
     const projectData = await executeGetProjectDataCommand(lsClient);
     KedroVizPanel.currentPanel?.updateData(projectData);
 }
+
+export async function isKedroProject(): Promise<boolean> {
+    const folders = vscode.workspace.workspaceFolders;
+    if (!folders) {
+      return false;
+    }
```

Comment:
Maybe use this function? https://github.com/kedro-org/vscode-kedro/blob/d0b3f9545734292a49ed2b3fb393bf8916125de0/src/common/vscodeapi.ts#L37

Reply:
Also have you considered the situation where multiple workspace exists? For now we don't have an option to choose the workspace/project if there are more than one. So the default is [using the first](https://github.com/kedro-org/vscode-kedro/blob/d0b3f9545734292a49ed2b3fb393bf8916125de0/src/common/utilities.ts#L55-L58)